src/main/java/org/spongepowered/common/datapack/SpongeDataPackManager.java

```diff
@@ -44,7 +48,8 @@
 @Singleton
 public final class SpongeDataPackManager {
 
-    public static SpongeDataPackManager INSTANCE;
+    // TODO @Zidane where do you want to instantiate this?
+    public static SpongeDataPackManager INSTANCE = new SpongeDataPackManager(Sponge.getGame());
```

Comment:
This can't be done like this. The class is Singleton which means that if it is injected elsewhere, it will be a different object ref.

So likely toss @Singleton then.